src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_PATTERN = "supply";
+    private static final String BUY_PATTERN = "buy";
+    private static final String RESULT_PATTERN = "result";
+    private static final int INDEX_OF_OPERATION = 0;
+    private static final int INDEX_OF_VALUE = 1;
+    private int supplyCounter = 0;
+    private int buyCounter = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String fileDataString = readFile(fromFileName);
+        countOperationsValue(fileDataString);
+        int result = supplyCounter - buyCounter;
+        writeFile(toFileName, new String[]{SUPPLY_PATTERN + "," + supplyCounter,
```

Comment:
lets create separate method for this data
something like createReport() and you can call countOperationsValue() inside